nio/client/async_client.py

```diff
@@ -1504,6 +1506,42 @@ async def delete_devices(
 
         return await self._send(DeleteDevicesResponse, method, path, data)
 
+    @logged_in_async
+    async def space_get_hierarchy(
+        self,
+        space_id: str,
+        from_page: Optional[str] = None,
+        limit: Optional[int] = None,
+        max_depth: Optional[int] = None,
+        suggested_only: bool = False,
+    ) -> Union[SpaceGetHierarchyResponse, SpaceGetHierarchyError]:
+        """Gets the space's room hierarchy.
+
+        Calls receive_response() to update the client state if necessary.
+
+        Returns either a `SpaceGetHierarchyResponse` if the request was successful
+        or a `SpaceGetHierarchyError` if there was an error with the request.
+
+        Args:
+            space_id (str): The ID of the space to get the hierarchy for.
```

Comment:
Yeah I agree! In fact, I initially had the `space_id` arg named as `room_id` but figured it might be better to just call it `space_id`. Maybe we should just call it room_id anyways since the outline of the request in the spec also uses room id: https://spec.matrix.org/latest/client-server-api/#get_matrixclientv1roomsroomidhierarchy

The description also could simply be:

```suggestion
            space_id (str): The room id of the space to get the hierarchy for.
```

or 

```suggestion
            space_room_id (str): The room id of the space to get the hierarchy for.
```